queries.sql

```diff
@@ -107,4 +111,24 @@ SELECT * from animals;
 
 Commit;
 
-SELECT * from animals
\ No newline at end of file
+SELECT * from animals
+
+-- ////Picture 4///
+
+SELECT count(*) from animals;
+
+SELECT count(*) from animals
+WHERE escape_attempts = 0;
+
+SELECT avg(weight_kg) from animals;
+
+SELECT neutered, count(*) as escape_attempts
+FROM animals
+WHERE escape_attempts > 0
+GROUP BY (neutered);
```

Comment:
- [ ] 👍 Great effort on writing your query! However, there's a small issue with the current query, and I'd like to help you make it even better. 

   - The goal of the query is to determine who escapes more frequently, either `neutered` or `not neutered` animals. To achieve this, you should modify the query to retrieve the results where `neutered` is `true` and `escape_attempts` is equal to `20`. 

Here's an example of a refined version of the query you can use to get the desired results:

```
SELECT neutered, SUM(escape_attempts) AS total_escapes
	FROM animals
	GROUP BY neutered
	ORDER BY total_escapes DESC
	LIMIT 1;
```

Happy coding! 😊👏